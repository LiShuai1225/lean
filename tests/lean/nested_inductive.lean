-- Error: cannot have type Type.{1} and Type.{0}
inductive foo (A B : Prop) : Prop
| mk : list foo -> foo
