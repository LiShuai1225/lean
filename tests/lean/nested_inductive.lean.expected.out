nested_inductive.lean:2:0: error: type mismatch at definition '23.foo', has type
  Prop → Prop → Type₁
but is expected to have type
  Prop → Prop → Prop
